phoenix-core/src/main/java/org/apache/phoenix/query/QueryServices.java

```diff
@@ -313,6 +313,9 @@ public interface QueryServices extends SQLCloseable {
     //Update Cache Frequency default config attribute
     public static final String DEFAULT_UPDATE_CACHE_FREQUENCY_ATRRIB  = "phoenix.default.update.cache.frequency";
 
+    // whether to validate last ddl timestamps during client operations
+    public static final String LAST_DDL_TIMESTAMP_VALIDATION_ENABLED = "phoenix.ddl.timestamp.validation.enabled";
```

Comment:
can we make it true at BaseTest level so that it applies to all tests? that way we will get more confidence?

Reply:
@virajjasani that makes sense, I can try running all tests with this property set.